napari/_qt/qt_viewer.py

```diff
@@ -278,18 +278,7 @@ def __init__(
             self.viewer.layers, self._qt_poll
         )
 
-        # moved from the old layerlist... still feels misplaced.
-        # can you help me move this elsewhere?
-        if config.async_loading:
-            from napari._qt.experimental.qt_chunk_receiver import (
-                QtChunkReceiver,
-            )
-
-            # The QtChunkReceiver object allows the ChunkLoader to pass newly
-            # loaded chunks to the layers that requested them.
-            self.chunk_receiver = QtChunkReceiver(self.layers)
-        else:
-            self.chunk_receiver = None
+        self.chunk_receiver = None
```

Comment:
As it stands, this overwrites the defined function that returns `None` with `None` itself.